src/lib/services/project-service.ts

```diff
@@ -57,6 +59,20 @@ export interface AccessWithRoles {
     groups: IGroupModelWithProjectRole[];
 }
 
+export interface IStatusUpdate {
+    avgTimeToProdCurrentWindow: number;
```

Comment:
is the number days/hours/minutes/ms? maybe type alias (tiny types) would help?
e.g. type Hours = number;